doc/self-hosted/index.md

```diff
@@ -38,18 +38,20 @@ USAGE
   $ code-server [WORKDIR]
 
 ARGUMENTS
-  WORKDIR  [default: (directory to binary)] Specify working dir
+  WORKDIR  [default: /root/project] Specify working dir
```

Comment:
This is only the default for the docker container.

Reply:
Yeah I just noticed it, I misread my diff, fixing it now.

Reply:
Resolved with a force-push to my branch.